scripts/publish-docker-images

```diff
@@ -127,12 +128,22 @@ push_multi-arch_manifest_docker_hub() {
 		rm multi-arch-temp.yaml
 	done
 
+	# Create digest files
+	if [[ ! -z "${FILE_PATH}" ]]; then
+			echo "Creating digest files"
+			{
+				for v in ${IMAGE_TAG_SHA} ${IMAGE_TAG_SHA_ARM} ${IMAGE_TAG_SHA_AMD}; do
+					./manifest-tool inspect --raw "${IMAGE_NAME}:${v}" | jq "{\"${IMAGE_NAME}:${v}\":.}";
+				done;
+			} | jq -s add > "${FILE_PATH}"
+			echo "Digest file created"
+	fi
+
 	rm -rf ${manifest_tool_dir}
```

Comment:
Wait a sec, I don't see a `cd` or a `popd` prior to this `rm`, so the dir removed is not actually where the `manifest-tool` is cloned to. 

Are we running into issues with this script as used today? I have a feeling we're not as we tend to use `mktemp` liberally. The existing management of the directory could become an issue if the caller's usage were to change.

This could be fixed by either using a `mktemp -d` where `manifest-tool` is cloned into, built, and used from (necessitating something like `make -C "$manifest_tool_dir"`, `"manifest_tool="$manifest_tool_dir/manifest-tool"`, `"$manifest_tool" inspect --raw ...`). As an added bonus, the `git clone` could be simplified to: `git clone --depth 1 --branch v0.9.0 https://github.com/estesp/manifest-tool "$manifest_tool_dir"` which avoids unnecessary network traffic, additional commands to checkout the rev, and leaves the repo ready to rumble!

If we don't want to go that route, the `manifest-tool` directory /could/ remain in its current place and simply use `pushd "$manifest_tool_dir"; $do_work; popd`, but I tend to prefer temp files :smile: 

Either way, take another read over this function to see if you see other broken assumptions or issues with using a temp dir - nicely enough, `manifest-tool` is fairly well scoped to this function so its pretty quick work.

Reply:
Added a `popd` before `rm -rf "${manifest_tool_dir}"`.